.github/workflows/pr_push_workflow.yml

```diff
@@ -0,0 +1,58 @@
+name: PR/push workflow
```

Comment:
`workflow` part in the name is rather redundant, if you want to stick to this name.

Reply:
Done

Reply:
Done